examples/post_training_quantization/tensorflow/mobilenet_v2/main.py

```diff
@@ -146,16 +145,16 @@ def transform_fn(data_item):
 ###############################################################################
 # Benchmark performance, calculate compression rate and validate accuracy
 
-ov_model = mo.convert_model(tf_model)
-ov_quantized_model = mo.convert_model(tf_quantized_model)
+ov_model = ov.convert_model(tf_model, share_weights=False)
+ov_quantized_model = ov.convert_model(tf_quantized_model, share_weights=False)
```

Comment:
Why the `share_weights` option was set to `False`?

Reply:
Good question! openvino.convert_model works incorrect for  share_weights=True (ref 126201)